xlators/mount/fuse/src/fuse-bridge.c

```diff
@@ -981,7 +1067,7 @@ send_fuse_err(xlator_t *this, fuse_in_header_t *finh, int error)
     if (inode)
         inode_unref(inode);
 
-    return send_fuse_iov(this, finh, &iov_out, 1);
+    return send_fuse_iov(this, finh, &iov_out, 1, true);
```

Comment:
Why this is sent synchronously ?

Reply:
In case send_fuse_err(), keeping in sync. When there is no memory and fuse wants to send err to /dev/fuse, sending the error in async mode required memory to be allocated which will again fail. So preferring sync mode for error case